### DISCUSSION TITLE
How can I apply conditional formatting in atoti UI?

### DISCUSSION THREAD

**Discussion by GameTn** [atoti/atoti#520] [Q&A] [answered]
Hello, 
Does atoti have any function to set a background color based on condition ?

**Comment by bloussou** [comment, marked as answer]
Hello,
As explain here (https://github.com/atoti/atoti/issues/367) in notebooks you can add some attributes to the MDX and a better way to do it will be available in the next release (a style menu will be available to do this properly.
![image](https://user-images.githubusercontent.com/32448554/158769616-f1378d24-3411-43c8-9d2b-48d60fb076a3.png)

**Reply by GameTn** [reply]
when is the next release? and will this affect my dashboard i'm working on it ?

**Reply by bloussou** [reply]
The next release date is not set yet (probably next week).
It won't be a breaking release so it will not break the dashboard you are working on.

**Reply by GameTn** [reply]
hi blousoou , i tried with MDX editor but i got no changes i did this code
```
With Member [Measures].[Salary1.SUM] As [Measures].[Salary.SUM] , BACK_COLOR = CASE
WHEN [Measures].[Salary1.SUM] > 1000 THEN rgb(124,252,0) END, CAPTION = [Measures].[Salary.SUM].MEMBER_CAPTION
```

**Reply by bloussou** [reply]
Did you try other values for the condition (`"1000"` instead of `1000`) not sure about this but you can try

**Reply by GameTn** [reply]
yea nothing changes also , even when i tried to set the text is visible in a pie chart with `isTextVisible:true`

**Reply by LaGuiche55** [reply]
Hi,

Continuing on this discussion and my initial request. Now I can modify the back ground color, thank you for this.
I think it would be great to have by default a conditional formatting of the font color (not the background color) similar to the Excel > Format Cells > Number > -1234.10 in red color.

Best
JP

![image](https://user-images.githubusercontent.com/43729246/171149954-acefc0ff-0e4c-43a3-b4e5-99309ceb2a57.png)
